tests/test_cliCmdQueryCollections.py

```diff
@@ -0,0 +1,72 @@
+# This file is part of daf_butler.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Unit tests for daf_butler CLI query-collections command.
+"""
+
+import click
+import unittest
+import yaml
+
+from lsst.daf.butler import Butler
+from lsst.daf.butler.cli.butler import cli
+from lsst.daf.butler.tests.mockeredTest import MockeredTestBase
+
+
+class QueryCollectionsCmdTest(MockeredTestBase):
+
+    defaultExpected = dict(repo=None,
+                           collection_type=None,
+                           flatten_chains=False,
+                           include_chains=None)
+
+    def test_minimal(self):
+        """Test only the required parameters, and omit the optional parameters.
+        """
+        self.run_test(["query-collections", "here"],
+                      self.makeExpected(repo="here"))
+
+    def test_all(self):
+        """Test all parameters"""
+        self.run_test(["query-collections", "here",
+                       "--flatten-chains",
+                       "--include-chains"],
+                      self.makeExpected(repo="here",
+                                        flatten_chains=True,
+                                        include_chains=True))
+
+
+class QueryCollectionsScriptTest(unittest.TestCase):
+
+    def testGetCollections(self):
+        run = "ingest/run"
+        tag = "ingest"
+        expected = {"collections": ["ingest/run", "ingest"]}
+        runner = click.testing.CliRunner()
+        with runner.isolated_filesystem():
+            _ = Butler.makeRepo("here")
+            _ = Butler("here", run=run, tags=[tag], collections=[tag])
```

Comment:
Can you add a comment here to say that the purpose of this call is to create some collections?